packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -559,6 +569,19 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
     this.$.comboBox.pageSize = this.pageSize;
   }
 
+  /** @private */
+  _placeholderChanged(placeholder) {
```

Comment:
Testing with the component on `/dev`, I couldn't see the placeholder when I added either through attribute or property. Checking it a bit further, I found out that the `_selectedItemsChanged` observer runs before `_placeholderChanged` and there the placeholder is changed to what's stored in `this.__savedPlaceholder`, which by the time of the execution is `undefined`, therefore effectively removing the placeholder altogether.

Reply:
Good catch, thanks. Added the default empty string value for the placeholder - for some reason, this triggers the observer for the `placeholder` property set via attribute before the `selectedItems` observer is executed.